### DISCUSSION TITLE
Proposal for contribution: Testcontainers for OpenSearch

### DISCUSSION THREAD

**Discussion by martin-schaefer** [testcontainers/testcontainers-java#6787] [Ideas]
Hi,

[OpenSearch](https://opensearch.org/) is getting more and more trajectory and is also adopted in my company. I have develop a TestContainer for the official [Docker image](https://hub.docker.com/r/opensearchproject/opensearch) which I would like to contribute and maintain.
Are you interested and how should I start the contribution?

**Comment by eddumelendez** [comment]
HI @martin-schaefer, look at https://github.com/opensearch-project/opensearch-testcontainers

**Reply by martin-schaefer** [reply]
Thanks for the hint, didn't know this.
